src/mixins/boundary.js

```diff
@@ -0,0 +1,60 @@
+import * as THREE from "three";
+
+export const BoundaryMixin = (superclass) => class extends superclass {
+
+    constructor(config) {
+        super(config);
+        this.boundaryConditions = config.boundaryConditions || {};
+    }
+
+    /**
+     * Boundaries are drawn only if type is "bc1", "bc2", or "bc3".
```

Comment:
It's to be explained what these are - perhaps referring to QE source or replicating the information here (in case the original source will change)

Reply:
Done.